www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -20,20 +20,18 @@
 include_once __DIR__ . "/../../class/centreonLog.class.php";
 $centreonLog = new CentreonLog();
 
-try {
-    $pearDB->query(
-        "UPDATE `contact` SET `contact_autologin_key` = NULL WHERE `contact_autologin_key` =''"
-    );
-} catch (\PDOException $e) {
-    $centreonLog->insertLog(
-        2,
-        "UPGRADE : 19.10.2 Unable to set default contact_autologin_key"
-    );
-}
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 20.04.0-beta.1 : ';
```

Comment:
next time, use shorter variable name, like "$upgradeVersion"

Reply:
indeed, but this variable is intended to be used on each upgrade script, so I thought that it should be easy to find and pretty clear